runtime/src/lib.rs

```diff
@@ -244,10 +242,6 @@ impl frame_system::Config for Runtime {
     type Index = Index;
     /// The lookup mechanism to get account ID from whatever is passed in dispatchers.
     type Lookup = IdentityLookup<AccountId>;
```

Comment:
don't we want to use `Indices` for `Lookup` like in standalone chain https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR292

Reply:
@ltfschoen I am not sure. I haven't changed this. It was already there https://github.com/DataHighway-DHX/node/blob/ilya/parachain-update/runtime/src/lib.rs#L246 in existing parachain code.

Can you create a ticket and then we can discuss with @festelo and @cgroeschel for the same?

Reply:
added to #227